app/assets/stylesheets/homepage.css.scss

```diff
@@ -25,9 +29,9 @@
     color: white;
     font-size: 1.1em;
     text-decoration: none;
-    height: 76px;
+    height: 75px;
     display: block;
-    padding: 8px 12px 0 12px;
+    padding: 9px 12px 0 12px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `9px 12px 0` instead of `9px 12px 0 12px`